internal/cmd/keys.go

```diff
@@ -171,7 +181,7 @@ func newKeysListCmd() *cobra.Command {
 			for _, k := range keys {
 				rows = append(rows, row{
 					Name:              k.Name,
-					IssuedFor:         k.IssuedFor.String(),
+					IssuedFor:         machineNames[k.IssuedFor],
```

Comment:
I think in the case of `if options.MachineName != ""` we don't populate `machineNames`, so it's going to output the empty string in those cases, which I assume we don't want

Reply:
Oops, nope, I'm wrong. It does populate it.